### PR TITLE
refactor(components/forms): adding react virtualized to enumeration

### PR DESCRIPTION
**Pull request by jlame**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
not virtualization of rows.

**What is the new behavior?**
Usage of react virtualize in the Enumeration component

**Does this PR introduce a breaking change?**

- [x] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...
A new props is needed in order to fix the rowHeight

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

File: packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js [hunk 1]
```diff
@@ -309,6 +310,10 @@ class EnumerationWidget extends React.Component {
 		}
 	}
 
+	getItemHeight(isInEdit) {
+		return isInEdit ? 33 : 33;
```
Comment (talend-sromain) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: Export 33 to const ?

File: packages/components/stories/Enumeration.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ const props = {
 		onItemChange: action('itemEdit.onItemchange'),
 		onAbortItem: action('itemEdit.onCancel'),
 		onSelectItem: action('itemEdit.onSelect'),
+		getItemHeight: (isInEdit) => { return 33; },
```
Comment (talend-sromain) [inline packages/components/stories/Enumeration.js]: Export 33 to const ?

File: packages/components/src/Enumeration/Items/items.snapshot.test.js [hunk 3]
```diff
@@ -20,6 +23,7 @@ describe('Items', () => {
 				onSubmitItem: jest.fn(),
 				onAbortItem: jest.fn(),
 				onSelectItem: jest.fn(),
+				getItemHeight: () => 42,
```
Comment (talend-sromain) [inline packages/components/src/Enumeration/Items/items.snapshot.test.js]: Export to const ?

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://332.talend.surge.sh)

**Pull request merged**